models/inventory-entry/src/index.ts

```diff
@@ -1,6 +1,29 @@
-export * as InventoryEntryDraft from './inventory-entry-draft';
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatInventoryEntryModelBuilder,
+} from './builders';
+import * as modelPresets from './presets';
+
+export * from './inventory-entry-draft';
 export * as InventoryEntry from '.';
 
-export { default as random } from './builder';
-export { default as presets } from './presets';
 export * from './types';
+
+export const InventoryEntryRest = {
+  random: RestModelBuilder,
+  presets: modelPresets.restPresets,
+};
+
+export const InventoryEntryGraphql = {
+  random: GraphqlModelBuilder,
+  presets: modelPresets.graphqlPresets,
+};
+
+/**
+ * @deprecated Use `InventoryEntryRest` or `InventoryEntryGraphql` exported models instead of `InventoryEntry`.
+ */
+export const deprecatedInventoryEntry = {
```

Comment:
Originally I had this looking like the doc example but I was getting this error when trying to commit:

models/inventory-entry/src/index.ts:8:1
  ✖   8:1   Multiple exports of name random.   import/export
  ✖   8:1   Multiple exports of name presets.  import/export
  ✖  26:14  Multiple exports of name random.   import/export
  ✖  27:14  Multiple exports of name presets.  import/export
  

So I changed it to this and that made it happier. Let me know if that's not correct though.

Reply:
I think that was because the deraft model has not been migrated yet.